Dockerfile

```diff
@@ -1,18 +1,5 @@
 # Build the manager binary
 FROM golang:1.17 as builder
-RUN apt-get update && apt-get install -y ca-certificates openssl
```

Comment:
@mgaldino-splunk Is this a change required for all environments? I understand you were may have needed this on your local system?